### DISCUSSION TITLE
How to refer to a certain KnowledgeBase in ollama ?

### DISCUSSION THREAD

**Discussion by mahmah2** [awaescher/OllamaSharp#140] [Q&A] [answered]
I have setup a knowledge base in Ollama using OpenWebUI interface and given it a name.

When using OpenWebUI prompt you can use a # sign to refer to a certain knowledge base and ask questions in that context.

I tried to add a # knowledge base at the beginning of the user query text but it seems that it doesn't work.

How can we set the context of the question to a certain knowledge base ?

**Comment by awaescher** [comment, marked as answer]
This is not possible because with OllamaSharp you are using Ollama directly. Data stored elsewhere - like in this case in OpenWebUI - is not accessible this way.

**Reply by mahmah2** [reply]
Thanks mate. You are right. Sorry for confusion.
